conf/airframes/tudelft/delfly_lisas.xml

```diff
@@ -0,0 +1,257 @@
+     flight_plan="flight_plans/rotorcraft_basic.xml"
+     settings="settings/rotorcraft_basic.xml settings/control/rotorcraft_guidance.xml settings/control/stabilization_att_int.xml ../conf/modules/logger_sd_spi_direct.xml (if SD logger is used)"
+-->
+
+<airframe name="delfly_lisa_s">
+
+  <!-- FIRMWARE %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%-->
+
+  <firmware name="rotorcraft">
+    <target name="ap" board="lisa_s_1.0">
+      <!-- SuperbitRF RC -->
+        <!-- module name="radio_control" type="superbitrf_rc"/-->
+      <!-- DelTang Rx31 -->
+      <module name="radio_control" type="ppm">
+        <configure name="RADIO_CONTROL_PPM_PIN" value="SERVO6"/>
+        <define name="RADIO_KILL_SWITCH" value="RADIO_GEAR"/>
+            <define name="RADIO_YAW" value="RADIO_ROLL"/>
+          </module>
+
+      <configure name="USE_MAGNETOMETER" value="0"/>
+      <configure name="AHRS_PROPAGATE_FREQUENCY" value="500"/>
+      <define name="USE_PERSISTENT_SETTINGS" value="TRUE"/>
+    </target>
+
+    <!-- Necessary to build succesfully, but not used -->
+    <module name="motor_mixing"/>
+
+    <!-- Rpm sensing settings (cannot be pushed to master) -->
+      <!--subsystem name="rpm_sensor" type="eagletree"-->
+        <!--define name="PULSES_PER_ROTATION" value="5"/-->
+      <!--/subsystem-->
+
+    <module name="actuators" type="pwm">
+      <define name="SERVO_HZ" value="400"/>
+      <define name="USE_SERVOS_1AND2"/>
+    </module>
+
+    <!-- TELEMETRY -->
+    <!-- SuperbitRF telemetry -->
+      <!--module name="telemetry" type="superbitrf"/-->
+    <!-- Debug serial interface telemetry -->
+    <module name="telemetry" type="transparent">
+      <configure name="MODEM_BAUD" value="B115200"/>
+      <configure name="MODEM_PORT" value="UART1"/>
+    </module>
+
+    <module name="imu" type="lisa_s_v1.0"/>
+    <module name="gps" type="ublox"/>
+    <module name="stabilization" type="int_euler" prefix="STABILIZATION_">
+      <define name="FILTER_CMD_ROLL_PITCH" value="1"/>
+      <define name="FILTER_CMD_YAW" value="1"/>
+      <define name="FILTER_CMD_ROLL_CUTOFF" value="15.0"/>
+      <define name="FILTER_CMD_PITCH_CUTOFF" value="15.0"/>
+      <define name="FILTER_CMD_YAW_CUTOFF" value="10.0"/>
+      <define name="ATTITUDE_SP_PSI_DELTA_LIMIT" value="1.57"/>
+    </module>
+    <module name="ahrs" type="int_cmpl_quat"/>
+    <module name="ins"/>
+  </firmware>
+
+  <!-- MODULES %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%-->
+
+  <modules main_freq="512">
+    <module name="send_imu_mag_current"/>
+    <!-- When sd logger is used, then radio should be DelTang (PPM) and telemetry should
+      be transparent (cable/WiFi) -->
+    <module name="logger_sd_spi_direct.xml">
+      <define name="SDLOGGER_ON_ARM" value="TRUE"/>
+      <!--define name="LOGGER_LED" value="3"/-->
+    </module>
+  </modules>
```

Comment:
We are moving away from using a separate modules section, can you move to contents of this to the firmware section above?